packages/node-core/src/meta/health.service.ts

```diff
@@ -39,7 +42,7 @@ export class HealthService {
     }
 
     if (healthy !== this.indexerHealthy) {
-      await this.storeService.storeCache.metadata.model.upsert({key: 'indexerHealthy', value: healthy});
+      await this.storeService.modelProvider.metadata.set('indexerHealthy', healthy);
```

Comment:
This used the model directly because we don't want this to be cached and add delay.